packages/ui/components/EditorSwitch.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import * as Switch from '@radix-ui/react-switch'
```

Comment:
@KhudaDad414 @fmvilas Radix-ui need an ADR ? or this was already decided in the past ?

Reply:
I saw a few radix-ui components being used so I don't know 😆

Reply:
I introduced it a few weeks ago so yeah, an ADR would be cool.

Reply:
Created an issue for this: https://github.com/asyncapi/studio/issues/734